metomi/rose/etc/rose-meta/fcm_make/HEAD/rose-meta.conf

```diff
@@ -0,0 +1,47 @@
+[=mode]
```

Comment:
Is this Git not picking up on a file rename or have you been having fun writing fcm metadata?

Reply:
Do you not enjoy writing FCM metadata in your time off? 

(No, this is just git being odd.)

Reply:
> Do you not enjoy writing FCM metadata in your time off?

My secret hobby.